friends/src/App.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import FriendsList from './components/FriendsList';
+import Friend from './components/Friend';
+import logo from './logo.svg';
+import './App.css';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friendsData: [],
+      name: '', 
+      age: '',
+      email: ''
+    };
+  }
+
+  componentDidMount() {
+    axios
+    .get("http://localhost:5000/friends")
+    .then(response => {
+      console.log("GET RESPONSE:", response);
+     this.setState({ friendsData: response.data  });
+    });
+  }
+
+  handleNameChange = e => {
+    this.setState({ [e.target.name]: e.target.value });
+  };
+
+  handleSubmitFriend = () => {
+    const friend = {friend: this.state.friend};
```

Comment:
I'm not sure that this would work as it is currently.  You are referencing this.state.friend,  but there is no "friend" on state.